css/layout.css

```diff
@@ -21,14 +21,23 @@
   flex-direction: column;
 }
 
-.row-body {
+.bm-contain {
+  max-width: 300px;
+  min-height: 190px;
   display: flex;
+  flex-direction: column;
 }
 
-#entry-list {
+.bm-modal-layout {
+  background-color: #d9d9d9;
+  padding: 2rem;
+  min-width: 364px;
+  min-height: 252px;
```

Comment:
You want to be very careful about using set `px` values. It prevents them from being fluid. I recommend percents or `rem`